index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microfolio</title>
+    <link
+      rel="shortcut icon"
+      href="https://img.icons8.com/external-others-inmotus-design/67/null/external-B-alphabet-others-inmotus-design-22.png"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <img id="menu" src="imgs/Union.svg" />
+    </nav>
+    <section id="content-1">
+      <h1>
+        <span class="clr-blue">Hey there. <br />I'm Blesslin <br /></span>
+        <span class="clr-green"> I'm a Software <br />Developer</span>
+      </h1>
+      <p class="content-1-p">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me.
+      </p>
```

Comment:
- [ ] [OPTIONAL] I highly suggest you check the `font-size` of the text on your page. It's looking big when compared to the Figma design. ✍️